api-report/container-utils.api.md

```diff
@@ -58,12 +58,15 @@ export class GenericError extends LoggingError implements IGenericError {
 
 // @public
 export class ThrottlingWarning extends LoggingError implements IThrottlingWarning {
```

Comment:
For any change in api.md, we should ask the question - is it breaking? And most important - does it belong here?
I believe the answer to second question is no, as there is only one consumer.
Can we move this class to container-loader package to reduce API surface?